week02/assignment02_test.go

```diff
@@ -0,0 +1,77 @@
+package week02_test
+
+import "testing"
+
+func TestArray(t *testing.T) {
+	exp := [4]string{"Golang", "Python", "JavaScript", "Ruby"}
+	var act [4]string
+
+	// copying the value
+	for i, v := range exp {
+		act[i] = v
+	}
+
+	for i, v := range act {
+		if exp[i] != v {
+			t.Errorf("Expected Value : %s \n Got value: %s", v, exp[i])
+		}
+	}
+}
+
+// test slicee
+
+func TestSlice(t *testing.T) {
+	exp := []string{"Docker", "Kubernetes", "Terraform", "Github Actions"}
+	var act []string
+
+	// copying the value
+	for _, v := range exp {
+		act = append(act, v)
+	}
+
+	for i, v := range act {
+
+		// checking length
+		if len(exp) != len(act) {
+			t.Errorf("Expected length: %d Got length %d", len(exp), len(act))
+
+		} else {
```

Comment:
In Go we try to limit `else` statements. This statement, for example, is unnecessary.